core/controllers/email_dashboard.py

```diff
@@ -48,7 +48,7 @@ class EmailDashboardPage(
 
     @acl_decorators.can_manage_email_dashboard
     def get(self) -> None:
-        """Handles GET requests."""
+        """Renders the email dashboard page."""
```

Comment:
I would rather prefer to say:
```suggestion
        """Decorator to render the email dashboard page."""
```

Because decorators are called a bit differently from regular functions.

Reply:
This is applicable to all decorators. If everyone agrees.